indico/htdocs/js/indico/jquery/global.js

```diff
@@ -61,8 +61,15 @@ $(document).ready(function() {
         var extraOpts = $(this).data('qtipOpts') || {},
             qtipClass = $(this).data('qtip-style');
 
-        $(this).qtip($.extend(true, {}, {
+        var target = $(this);
```

Comment:
Ok, I remember a comment from someone stating that I should always prefix jQuery objects with `$` (not only `$this`)